geometry_msgs/Pose[] detectedObjs